filer/admin/folderadmin.py

```diff
@@ -375,8 +375,30 @@ def directory_listing(self, request, folder_id=None, viewtype=None):
             .order_by("-modified")
         )
         file_qs = file_qs.annotate(
-            thumbnail_name=Subquery(thumbnail_qs.filter(name__contains=f"__{size}_").values_list("name")[:1]),
-            thumbnailx2_name=Subquery(thumbnail_qs.filter(name__contains=f"__{size_x2}_").values_list("name")[:1])
+            # For private files (Filer pattern)
+            thumbnail_name_filer=Subquery(
+                thumbnail_qs.filter(name__contains=f"__{size}_").values_list("name")[:1]
+            ),
+            thumbnailx2_name_filer=Subquery(
+                thumbnail_qs.filter(name__contains=f"__{size_x2}_").values_list("name")[
+                    :1
+                ]
+            ),
+            # For public files (easy_thumbnails pattern)
+            thumbnail_name_easy=Subquery(
+                thumbnail_qs.filter(name__contains=f".{size}_q85_crop").values_list(
```

Comment:
Indeed, `_q85`  or `crop` imho should not be part of the query. 

Can this all be simplified by customizing the part before `{size}` (either `.` or `__`). Also, how would reversability be affected if we also did use the `__` characters before size? Then this would dramatically simplify.